message_app/model.py

```diff
@@ -53,3 +53,9 @@ def select(cls, username: str) -> Union[User, None]:
         # TODO: Select multiple users with multiple conditions
         user = User.query.filter_by(username=username).first()
         return user
+
+    @classmethod
+    def select_all(cls) -> List:
+        all_users = User.query.all()
+        return all_users
```

Comment:
I do not think getting all users is the correct way to do what you are doing here. As a rule of thumb, we just fetch what we need. 
Here we just need the ID of the last user in the database, and when we add a new user, assert that the new user's ID is equals to last user's ID + 1
Something like this:
```
last_user_id = User.get_last().id # you will have to implement this
User.insert(new_user)
new_user.id == last_user_id
```

Reply:
@khuolo01 We should not create a new parameter just for the sake of testing. What I suggest here is using a SQLAlchemy command to achieve what we want here.
Please refer to this thread on StackOverflow: https://stackoverflow.com/questions/8551952/how-to-get-last-record